pod2daemon/Makefile

```diff
@@ -77,35 +77,22 @@ bin/flexvol-armv7: ARCH=armv7
 bin/flexvol-ppc64le: ARCH=ppc64le
 bin/flexvol-s390x: ARCH=s390x
 bin/flexvol-%: $(SRC_FILES)
-	$(DOCKER_RUN) -e CGO_ENABLED=0 $(CALICO_BUILD) go build -buildvcs=false -v -o bin/flexvol-$(ARCH) flexvol/flexvoldriver.go
+	$(call build_binary, flexvol/flexvoldriver.go, $@)
 
 $(BINDIR)/csi-driver-arm64: ARCH=arm64
 $(BINDIR)/csi-driver-armv7: ARCH=armv7
 $(BINDIR)/csi-driver-ppc64le: ARCH=ppc64le
 $(BINDIR)/csi-driver-s390x: ARCH=s390x
 $(BINDIR)/csi-driver-%: $(SRC_FILES)
-	$(DOCKER_RUN) -e CGO_ENABLED=$(CGO_ENABLED) $(CALICO_BUILD) go build -buildvcs=false -v -o $(BINDIR)/csi-driver-$(ARCH) csidriver/main.go
-
-$(BINDIR)/csi-driver-amd64: $(SRC_FILES)
-ifeq ($(FIPS), true)
-	$(call build_static_cgo_boring_binary, csidriver/main.go, $@)
-else
-	$(call build_static_binary, csidriver/main.go, $@)
-endif
+	$(call build_binary, csidriver/main.go, $@)
```

Comment:
I believe this csi driver talks to kubelet via unix domain socket and no encryption is involved so cgo is disabled.